app/main/controllers/template/rtmedia-functions.php

```diff
@@ -543,6 +543,9 @@ function rtmedia_media( $size_flag = true, $echo = true, $media_size = 'rt_media
 		if ( 'photo' === $rtmedia_media->media_type ) {
 			$src  = wp_get_attachment_image_src( $rtmedia_media->media_id, $media_size );
 
+			//Addtime stamp to resolve conflict with cache image.
+			$src[0] = $src[0] . '?' . time();
```

Comment:
Same as above add `'?' . time();` in `set_url_scheme` usage of `$html`

Reply:
>Add timestamp to resolve conflict with cache image.

add this as the last line of multiline comment, to indicate addition of timestamp.